.github/workflows/tests.yml

```diff
@@ -57,9 +57,9 @@ jobs:
         run: |
           coverage run --rcfile=setup.cfg -m pytest test/serial
 
-      - name: Run MPI tests (procs=2, 4, 7, 12)
+      - name: Run MPI tests (procs=2, 4, 7)
         run: |
-          PROCS=(2 4 7 12)
+          PROCS=(2 4 7)
```

Comment:
Dropping procs=12 reduces the time for the MPI tests by more than an hour. Is this because there are quite a few combinations we are testing against or from removing any synchronisations etc?